src/main.py

```diff
@@ -0,0 +1,15 @@
+from fastapi import FastAPI
+
+from src.services.routers.version import router as version_router
+
+app = FastAPI()
+
+# Include the version router
+app.include_router(version_router, prefix="/api")
+
+if __name__ == "__main__":
+    import uvicorn
+    from src.core.utils import ConfigLoader
+
+    properties = ConfigLoader.load_application_properties()
+    uvicorn.run(app, **properties)
```

Comment:
@vasylboliuk please put uvicorn and other imports at the top of the file. Then you can define main function, that will contain logic on lines 14-15 (please try not define it in the global scope, you will have problems with writing tests later on) and inder __name__ there will be only main() call.

Important: that is why your application code should be structured very carefully and incapsulated in some classes and modules - you don't want your application to get a circular import error or to run some code defined in the global scope of the modules you import from main.

You can also look into the wsgi and asgi approach to not use __name__ approach at all.

Reply:
Fixed